### DISCUSSION TITLE
downsampled data

### DISCUSSION THREAD

**Discussion by Jimyhjy** [thanos-io/thanos#4599] [Questions & Answers]
Hi.
    Why the downsampled data is still about the same size as the original data and will not decrease?

**Comment by yeya24** [comment]
It is possible but unlikely. Can you show the meta json files of the raw block and the downsampled block? 
For raw blocks and downsampled blocks, the index size should be the same but downsampled block should have smaller chunk sizes.

**Comment by bwplotka** [comment]
Hey, this doc should help understand why: https://thanos.io/tip/components/compact.md/#-downsampling-note-about-resolution-and-retention- (:

**Reply by Jimyhjy** [reply]
I've read this document, normally, if fewer data points are extracted in the same time period, the blocks should also be reduced, but why does downsampling not reduce disk space? Can you tell me the principle?
